capa/features/extractors/viv/insn.py

```diff
@@ -175,6 +175,11 @@ def derefs(vw, p):
     while True:
         if not vw.isValidPointer(p):
             return
+
+        if vw.isProbablyString(p) or vw.isProbablyUnicode(p):
+            # don't deref strings that coincidentally are pointers
+            return
```

Comment:
e.g. `ACS...` which would be `0x430041` (valid pointer) as seen in mimikatz at 0x493E30